content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -1,139 +1,152 @@
-
-In TaxCloud
------------
-* Create a free account on `TaxCloud
-  <https://taxcloud.com/#register>`__ website.
-* Register your Odoo website on TaxCloud to get an *API ID* and an *API Key*.
-
-.. image:: taxcloud/taxcloud01.png
-  :align: center
-
-* In Settings on TaxCloud, click *Locations* to enter the location of your Office(s) & Warehouse(s).
-* In Settings on TaxCloud, click *Manage Tax States* to verify the states where you collect sales tax.
-
-In Odoo
--------
-* Go to :menuselection:`Invoicing / Accounting --> Configuration --> Settings`
-  and check *TaxCloud - Compute tax rates based on U.S. ZIP codes* (note: actually uses complete street address).
-* Enter your TaxCloud credentials.
-* Click SAVE to store your credentials.
-
-.. image:: taxcloud/taxcloud02.png
-  :align: center
-
-* Click the Refresh Icon next to *Default Category* to import the TIC product categories
-  from TaxCloud (Taxability Information Codes). Some categories may imply specific rates or attract product/service type exemptions.
-* Select your default *TIC Code*. This will apply to any new
-  product created.
-* Set a specific TaxCloud TIC Category on the *General Information* tab of the Product,
-  or on the Product Category.
-* Make sure your company address is complete (including the state
-  and the zip code). Go to :menuselection:`Settings --> Users & Companies --> Companies`
-  to open and edit your Company record.
-
-Automatically post taxes to the correct Tax Payable account
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* For each one of your companies that uses TaxCloud, it might be necessary to create a 
-  **User-Defined Default** so that the new taxes generated by the TaxCloud integration are created 
-  with the correct Tax Payable account:
-  
-.. warning::
-   A User-Defined Default impacts all records at creation.  It means that **every** new tax will be 
-   set up to record income in the specified Tax Payable account, unless the tax is manually edited
-   to specify a different income account (or there exists another User-Defined Default that takes 
-   precedence).
+====================
+TaxCloud integration
+====================
 
-* In :menuselection:`Accounting --> Configuration --> Chart of Accounts`, select the Tax Payable
-  account for the company. Take note of the account's ``id`` in the URL string. 
+TaxCloud calculates the sales tax rate in real-time for every state, city, and special jurisdiction
+in the United States. It keeps track of which products are exempt from sales tax and in which states
+each exemption applies.
 
-.. image:: taxcloud/user-default-find-account-id.png
-   :alt: The account's ID can be found in the URL string as 'id=...'. 
+TaxCloud registration
+=====================
+
+Register an account on `TaxCloud.com <https://taxcloud.com/register>`_ and complete the setup.
+Once you go live, get the :guilabel:`TaxCloud API Keys` by clicking on :guilabel:`Stores`, then
+:guilabel:`Get Details`.
```

Comment:
I don't see "get details"

Reply:
It's not shown on this screenshot. I show the resulting page where you find the API key.